apps/examples/src/examples.tsx

```diff
@@ -7,22 +7,99 @@ export type Example = {
 	path: string
 	codeUrl: string
 	hide: boolean
+	category: 'basic' | 'editor' | 'ui' | 'collaboration' | 'data/assets' | 'shapes/tools' | null
 	order: number | null
 	componentFile: string
 	loadComponent: () => Promise<ComponentType>
 }
 
-export const examples = (
+const basicExamples = (
 	Object.values(import.meta.glob('./examples/*/README.md', { eager: true })) as Example[]
-).sort((a, b) => {
-	// sort by order then title:
-	if (a.order === b.order) {
-		return a.title.localeCompare(b.title)
-	} else if (a.order === null) {
-		return 1
-	} else if (b.order === null) {
-		return -1
-	} else {
-		return a.order - b.order
-	}
-})
+)
+	.filter((e) => e.category === 'basic')
+	.sort((a, b) => {
+		if (a.order === null) {
+			return 1
+		} else if (b.order === null) {
+			return -1
+		} else {
+			return a.order - b.order
+		}
+	})
+
+const uiExamples = (
+	Object.values(import.meta.glob('./examples/*/README.md', { eager: true })) as Example[]
+)
+	.filter((e) => e.category === 'ui')
+	.sort((a, b) => {
+		if (a.order === null) {
+			return 1
+		} else if (b.order === null) {
+			return -1
+		} else {
+			return a.order - b.order
+		}
+	})
+const shapeExamples = (
+	Object.values(import.meta.glob('./examples/*/README.md', { eager: true })) as Example[]
+)
+	.filter((e) => e.category === 'shapes/tools')
+	.sort((a, b) => {
+		if (a.order === null) {
+			return 1
+		} else if (b.order === null) {
+			return -1
+		} else {
+			return a.order - b.order
+		}
+	})
+
+const editorExamples = (
+	Object.values(import.meta.glob('./examples/*/README.md', { eager: true })) as Example[]
+)
+	.filter((e) => e.category === 'editor')
+	.sort((a, b) => {
+		if (a.order === null) {
+			return 1
+		} else if (b.order === null) {
+			return -1
+		} else {
+			return a.order - b.order
+		}
+	})
+const dataExamples = (
+	Object.values(import.meta.glob('./examples/*/README.md', { eager: true })) as Example[]
+)
+	.filter((e) => e.category === 'data/assets')
+	.sort((a, b) => {
+		if (a.order === null) {
+			return 1
+		} else if (b.order === null) {
+			return -1
+		} else {
+			return a.order - b.order
+		}
+	})
+
+const collaborationExamples = (
+	Object.values(import.meta.glob('./examples/*/README.md', { eager: true })) as Example[]
+)
+	.filter((e) => e.category === 'collaboration')
+	.sort((a, b) => {
+		if (a.order === null) {
+			return 1
+		} else if (b.order === null) {
+			return -1
+		} else {
+			return a.order - b.order
+		}
+	})
+
+export const examples = [
+	{ id: 'Getting Started', array: basicExamples },
+	{ id: 'UI/Theming', array: uiExamples },
```

Comment:
this is touched upon above but `array` is too specific on what basicExamples is. it might be that basicExamples becomes an object in the future. a better wording here would be `values` because it might change.